hooks/tinted-jqp.sh

```diff
@@ -22,10 +22,6 @@ if [[ -d "$TINTED_JQP_CONFIG_FILE" ]]; then
   return 1
 fi
 
-if [ ! -e "$TINTED_JQP_CONFIG_FILE" ]; then
-  touch "$TINTED_JQP_CONFIG_FILE"
-fi
```

Comment:
Not necessary as the theme files from `tinted-jqp` are just copied over in-place.